src/pwa-buildpack/project-setup/create-configuration-files/index.md

```diff
@@ -0,0 +1,249 @@
+---
+title: Create configuration files
+---
+
+In the [previous topic], you created a `package.json` file and installed the project dependencies.
+In this topic, you will create the other configuration files for your development environment.
+
+## Create the Babel configuration file
+Create a `.babelrc` file in your theme’s root directory with the following content:
+
+``` javascript
+{
+    "plugins": [
+        "syntax-jsx",
+        "transform-class-properties",
+        "transform-object-rest-spread",
+        "transform-react-jsx"
+    ]
+}
+```
+
+This Babel configuration file allows you to use [JSX] in your project.
+JSX allows you to write React components using a syntax structure similar to HTML.
+
+See [Introducing JSX] for more information.
+
+## Create the local environment variables file
+
+Create a `.env` file and assign values to the following environment variables:
+
+* `MAGENTO_BACKEND_DOMAIN` - Your local Magento store’s host and port.
+* `MAGENTO_PATH` - The absolute path to the root directory of your local Magento store
+* `MAGENTO_BACKEND_PUBLIC_PATH` - Your theme’s vendor and name. The locale must be `en_US` for now.
+* `SERVICE_WORKER_FILE_NAME` - Set this value to `"sw.js"`.
+
+Your file should look like the following:
+
+``` text
+MAGENTO_BACKEND_DOMAIN=https://localhost.magento:8008
+
+MAGENTO_PATH=~/path/to/magento/rootdir
+
+MAGENTO_BACKEND_PUBLIC_PATH=/pub/static/frontend/OrangeCompany/orange-theme/en_US
+
+SERVICE_WORKER_FILE_NAME="sw.js"
+```
+
+## Create the webpack configuration file
+
+Create a `webpack.config.js` file in your theme’s root directory.
+This file exports a configuration object that tells Webpack how to build your theme.
+
+Webpack configuration is a complicated topic, so the following sections explain what each piece does in the configuration file.
+To see the fully assembled file, see the example [webpack.config.js].
+
+### Import environment variables
+
+At the top of the webpack.config.js file add the following:
+
+``` javascript
+require(`dotenv`).config();
```

Comment:
Better change the backticks to single quotes here. It should be `require('dotenv').config();`